onnxruntime/core/platform/apple/logging/apple_log_sink.mm

```diff
@@ -15,7 +15,9 @@
 void AppleLogSink::SendImpl(const Timestamp& timestamp, const std::string& logger_id, const Capture& message) {
```

Comment:
The file still includes the third-party "date.h" header file, even when timestamp_ns refers to std::chrono